docs/samples/specification/directives/generated/azure/directives/sample/_polling_paging_example.py

```diff
@@ -44,6 +44,9 @@ def __init__(
         self._deserialize = Deserializer(client_models)
 
 
+    def invoke(self, request, **kwargs):
+        return self._client._pipeline.run(request, stream=False, **kwargs)
```

Comment:
I'm ok to put "stream=False" as default, but if you do it that way it can't be overriden by kwargs (since I would get a double kwargs error). You need something like:

```python
stream = kwargs.get("stream", False)
```